norse/torch/functional/izhikevich.py

```diff
@@ -0,0 +1,159 @@
+    return s, p
+
+
+def subthreshold_oscillation(shape) -> Tuple[IzhikevichState, IzhikevichParameters]:
+    p = IzhikevichParameters(a=0.05, b=0.26, c=-60, d=0, tau_inv=250)
+    s = IzhikevichState(v=-62 * torch.ones(shape), u=-62 * p.b * torch.ones(shape))
+    return s, p
+
+
+def resonator(shape) -> Tuple[IzhikevichState, IzhikevichParameters]:
+    p = IzhikevichParameters(a=0.1, b=0.26, c=-60, d=-1, tau_inv=250)
+    s = IzhikevichState(v=-62 * torch.ones(shape), u=-62 * p.b * torch.ones(shape))
+    return s, p
+
+
+def integrator(shape) -> Tuple[IzhikevichState, IzhikevichParameters]:
+    p = IzhikevichParameters(a=0.02, b=-0.1, c=-55, d=6, mn=4.1, bias=108, tau_inv=250)
+    s = IzhikevichState(v=-60 * torch.ones(shape), u=-60 * p.b * torch.ones(shape))
+    return s, p
+
+
+def rebound_spike(shape) -> Tuple[IzhikevichState, IzhikevichParameters]:
+    p = IzhikevichParameters(a=0.03, b=0.25, c=-60, d=4, tau_inv=200)
+    s = IzhikevichState(v=-64 * torch.ones(shape), u=-64 * p.b * torch.ones(shape))
+    return s, p
+
+
+def rebound_burst(shape) -> Tuple[IzhikevichState, IzhikevichParameters]:
+    p = IzhikevichParameters(a=0.03, b=0.25, c=-52, d=0, tau_inv=200)
+    s = IzhikevichState(v=-64 * torch.ones(shape), u=-64 * p.b * torch.ones(shape))
+    return s, p
+
+
+def threshhold_variability(shape) -> Tuple[IzhikevichState, IzhikevichParameters]:
+    p = IzhikevichParameters(a=0.03, b=0.25, c=-60, d=4, tau_inv=250)
+    s = IzhikevichState(v=-64 * torch.ones(shape), u=-64 * p.b * torch.ones(shape))
+    return s, p
+
+
+def bistability(shape) -> Tuple[IzhikevichState, IzhikevichParameters]:
+    p = IzhikevichParameters(a=0.1, b=0.26, c=-60, d=0, tau_inv=250)
+    s = IzhikevichState(v=-61 * torch.ones(shape), u=-61 * p.b * torch.ones(shape))
+    return s, p
+
+
+def dap(shape) -> Tuple[IzhikevichState, IzhikevichParameters]:
+    p = IzhikevichParameters(a=1.0, b=0.2, c=-60, d=-21, tau_inv=100)
+    s = IzhikevichState(v=-70 * torch.ones(shape), u=-70 * p.b * torch.ones(shape))
+    return s, p
+
+
+def accomodation(shape) -> Tuple[IzhikevichState, IzhikevichParameters]:
+    p = IzhikevichParameters(a=0.02, b=1.0, c=-55, d=4, tau_inv=500)
+    s = IzhikevichState(v=-65 * torch.ones(shape), u=-16 * torch.ones(shape))
+    return s, p
+
+
+def inhibition_induced_spiking(shape) -> Tuple[IzhikevichState, IzhikevichParameters]:
+    p = IzhikevichParameters(a=-0.02, b=-1.0, c=-60, d=8, tau_inv=250)
+    s = IzhikevichState(v=-63.8 * torch.ones(shape), u=-63.8 * p.b * torch.ones(shape))
+    return s, p
+
+
+def inhibition_induced_bursting(shape) -> Tuple[IzhikevichState, IzhikevichParameters]:
+    p = IzhikevichParameters(a=-0.026, b=-1.0, c=-45, d=-2, tau_inv=250)
+    s = IzhikevichState(v=-63.8 * torch.ones(shape), u=-63.8 * p.b * torch.ones(shape))
+    return s, p
+
+
+def izhikevich_step(
+    input_current: torch.Tensor,
+    s: IzhikevichState,
+    p: IzhikevichParameters,
+    dt: float = 0.001,
+):
```

Comment:
A few lines of documentation would be really helpful. Perhaps just to describe what the model is and where to read more about it.